src/components/commons/Epoch/FirstEpoch/index.tsx

```diff
@@ -130,7 +130,7 @@ export default function FirstEpoch({ data: currentEpochData, onClick }: IProps)
       ),
       value: (
         <Content data-testid="epoch.firstEpoch.slotValue">
-          {moment(formatDateTimeLocal(currentEpochData.endTime)).diff(moment()) >= 0
+          {moment(formatDateTimeLocal(currentEpochData.endTime), "YYYY-MM-DDTHH:mm:ssZ").diff(moment()) >= 0
```

Comment:
Since we are using this date format everywhere. Please extract it to a constant and use this constant everywhere. This makes it easier to change it, if needed.